ga4gh/testbed/submit/report_submitter.py

```diff
@@ -0,0 +1,19 @@
+import requests
+
+
+class ReportSubmitter():
+
+    def submit_report(series_id, series_token, report, url="http://localhost:4500/reports"):
+        '''
+        Submits a report to the GA4GH testbed api.
+        
+        Required arguments:
+            series_id - A series ID is needed by server to group the report
+            series_token - A token is needed to verify authenticity
+            report - GA4GH report in JSON format
+            url - URL of the testbed server
+        '''
+        header = {"GA4GH-TestbedReportSeriesId": series_id, "GA4GH-TestbedReportSeriesToken": series_token}
+        submit_request = requests.post(url, headers=header ,json=report)
+        return submit_request.status_code
```

Comment:
Returning the status code from this method is insufficient, especially if there's been an error we want to know why (i.e. the message). If the submission has been successful, then we want the `id` of the new report that was created on the server. So,

- Always return status code
- Return error message if submission wasn't successful, `None` if submission was successful
- Return new report id if submission was successful, `None` if submission wasn't successful

Should be pretty easy to do in a Python dictionary